modules/apps/foundation/roles/roles-admin-web/src/main/java/com/liferay/roles/admin/web/internal/portlet/RolesAdminPortlet.java

```diff
@@ -282,6 +283,11 @@ public void updateActions(
 
 		long roleId = ParamUtil.getLong(actionRequest, "roleId");
 
+		long groupId = themeDisplay.getScopeGroupId();
+
+		_permissionService.checkPermission(
+			groupId, Role.class.getName(), roleId);
```

Comment:
Sorry, what do you mean by `RemoteLocalService`?  Do you mean `ResourcePermissionService`?  Or is there another class I can use to check permissions apart from the relevant service?